flink-table/flink-table-planner/src/test/scala/org/apache/flink/table/planner/runtime/batch/sql/FileSystemTestCsvITCase.scala

```diff
@@ -32,12 +35,11 @@ class FileSystemTestCsvITCase extends BatchFileSystemITCaseBase {
 
   override def getScheme: String = "test"
 
-  @After
+  @AfterEach
   def close(): Unit = {
     val path = new Path(resultPath)
-    assertEquals(
-      s"File $resultPath is not closed",
-      0,
-      TestFileSystem.getNumberOfUnclosedOutputStream(path))
+    if (TestFileSystem.getNumberOfUnclosedOutputStream(path) != 0) {
+      Assertions.fail(s"File $resultPath is not closed");
+    }
```

Comment:
When I tried using `Assertions.assertThat (xxx).as(yyy).isEqualTo(zzz)`, I found that there seemed to be some bugs that caused the IDE to think it couldn't compile. This is indeed very strange, causing me to have to deal with it this way. Do you have any other ideas?

Reply:
yeah.. that's weird...
there is WA for that
```scala
    assertThat(TestFileSystem.getNumberOfUnclosedOutputStream(path))
      .as(s"File $resultPath is not closed")
      .asInstanceOf[AbstractIntegerAssert[_]]
      .isEqualTo(0)
```

how there is an issue with such WA: in case there is more than one such WA per class it leads to issues for IntellijIdea (mvn works fine)...
I've files scala plugin issue https://youtrack.jetbrains.com/issue/SCL-20679

Reply:
I spent some time trying something similar, but I didn't come to any useful conclusions. Feeling disappointed about this 😞.